--- event.js	2009-02-19 10:52:00.000000000 +0100
+++ event_patched.js	2009-06-25 16:27:32.000000000 +0200
@@ -716,7 +716,7 @@
              * @private
              */
             startInterval: function() {
-                if (!this._interval) {
+                if (this._interval == null) {
                     var self = this;
                     var callback = function() { self._tryPreloadAttach(); };
                     this._interval = setInterval(callback, this.POLL_INTERVAL);
@@ -1656,7 +1656,7 @@
 
                 if (onAvailStack.length === 0) {
                     retryCount = 0;
-                    if (this._interval) {
+                    if (this._interval != null) {
                         clearInterval(this._interval);
                         this._interval = null;
                     } 
@@ -1746,7 +1746,7 @@
 
                     this.startInterval();
                 } else {
-                    if (this._interval) {
+                    if (this._interval != null) {
                         clearInterval(this._interval);
                         this._interval = null;
                     }
